docs/modules/publisher.md

```diff
@@ -121,24 +126,205 @@ A basic configuration file looks like this:
+| `HeartbeatIntervalTimespan`      | No        | String  | null                      | The interval used for the node to publish a value (a publisher <br>cached one) even if the value has not been <br>changed at the source. This value is represented in seconds. <br>Value expressed in Timespan string({d.hh:mm:dd.fff}).     |
+| `SkipFirst`                      | No        | Boolean | false                     | Instructs the publisher not to add to telemetry the<br> Initial DataChange (after subscription activation) for this OpcNode.     |
+| `QueueSize`                      | No        | Integer | 1                         | The desired QueueSize for the monitored item to be published.     |
+
+*Note*: **Id field may be omitted when ExpandedNodeIdId is present.
+
+A DataSet is a group of NodeIds within an OPC UA Server to be published with a same publishing interval.  
+DataSetWriter has one DataSet and contains the elements required to successfully establish a connection to the OPC UA Server. 
+A DatSetGroup is used to several DataSetWriters for a specific OPC UA server. 
+
+#### **PublishNodes_V1**
+
+PublishNodes enables a client to add a set of nodes to be published for a specific DataSetWriter. When a DataSetWriter already exist, the nodes shall be incrementally added to the very same dataset. When it does not already exist, a new DataSetWritter shall be created with the initial set of nodes contained in the request.
+  
+  *Request:* follows strictly the request payload schema as described above, the OpcNodes attribute being mandatory.
+
+  *Response:*
+
+  *Exceptions:*
+
+  *Example:*
+
+#### **UnpublishNodes_V1**
+
+UnpublishNodes method enables a client to remove nodes from a previously configured DataSetWriter.
+
+*Note*: If all the nodes from a DataSet are to be unpublished, the DataSetWriter entity shall be completely removed from the configuration storage.
+
+  *Request:*  follows strictly the request payload schema as described above, the OpcNodes attribute being mandatory.
+
+  *Response:*
+
+  *Exceptions:*
+
+  *Example:*
+
+#### **UnpublishAllNodes_V1**
+
+*TODO: add description*
+
+  *Request:*
+
+  *Response:*
+
+  *Exceptions:*
+
+  *Example:*
+
+#### **GetConfiguredEndpoints_V1**
+
+*TODO: add description*
+
+  *Request:*
+
+  *Response:*
+
+  *Exceptions:*
+
+  *Example:*
+
+#### **GetConfiguredNodesOnEndpoints_V1**
+
+*TODO: add description*
+
+  *Request:*
+
+  *Response:*
+
+  *Exceptions:*
+
+  *Example:*
+
+#### **GetDiagnosticInfo_V1**
+
+*TODO: add description*
+
+  *Request:*
+
+  *Response:*
+
+  *Exceptions:*
+
+  *Example:*
+
+#### **AddOrUpdateNodes_v1**
```

Comment:
Please rename `AddOrUpdateNodes_v1` to `AddOrUpdateEndpoints_V1`.

Reply:
done